apps/studio/components/interfaces/Database/Wrappers/WrappersDisabledState.tsx

```diff
@@ -24,34 +33,46 @@ const WrappersDisabledState = () => {
     'extensions'
   )
 
+  const { mutateAsync: enableExtension } = useDatabaseExtensionEnableMutation()
+
   const onEnableWrappers = async () => {
     if (wrappersExtension === undefined || vaultExtension === undefined) return
+    if (project === undefined) return console.error('Project is required')
+    if (project.connectionString === undefined)
+      return console.error('Connection string is required')
+
     setIsEnabling(true)
 
     const requiredExtensions = await Promise.all([
-      await meta.extensions.create({
+      await enableExtension({
+        projectRef: project.ref,
+        connectionString: project.connectionString,
         schema: wrappersExtension.schema ?? 'extensions',
         name: wrappersExtension.name,
         version: wrappersExtension.default_version,
         cascade: true,
       }),
-      await meta.extensions.create({
+      await enableExtension({
```

Comment:
```suggestion
      enableExtension({
```

Reply:
i've made it as a mutateAsync though, cause we need both extensions enabled first before wrappers is labelled as enabled

Reply:
ahh just saw your previous comment, was hidden haha